BLE_GattServer_FOTAService/README.md

```diff
@@ -0,0 +1,72 @@
+# FOTA Service Mock Example
+
+The FOTA service, as defined in its [specification](https://github.com/ARMmbed/mbed-os-experimental-ble-services/blob/fota-service-github-ci/services/FOTA/docs/README.md) document, facilities the transfer of firmware updates over BLE.
+
+In this demo, the FOTA service is used to transfer a binary from the host PC into the flash of the target. 
+A basic _FOTA client_, implemented in Python using [bleak](https://pypi.org/project/bleak/), is used to read/write the binary stream, control and status characteristics. 
+Please refer to [Section 1](https://github.com/ARMmbed/mbed-os-experimental-ble-services/tree/fota-service-github-ci/services/FOTA/docs#fota-service-structure) of the spec to learn more about these characteristics.
+
+To verify the success of the transfer, the application computes the [SHA-256](https://en.wikipedia.org/wiki/SHA-2) of the binary and prints it to the serial port.
+The SHA-256 is also computed on the host using [sha256sum](https://man7.org/linux/man-pages/man1/sha256sum.1.html).
+The transfer is regarded as a success if the hashes are equal.
+
```

Comment:
That would be a good thing to explain what is required to integrate the example on a real application and realize a real software update.